webui/react/src/pages/OldProjectDetails.tsx

```diff
@@ -733,7 +733,7 @@ const ProjectDetails: React.FC = () => {
       try {
         const result = await sendBatchActions(action);
         if (action === Action.OpenTensorBoard && result) {
-          openCommand(result as CommandTask);
+          openCommandResponse(result as CommandResponse);
```

Comment:
`result` can have `CommandResponse | void[] | CommandTask`.
Is it guaranteed that type of `result` here is always `CommandResponse`?
If not, i think its safer to narrow type https://www.typescriptlang.org/docs/handbook/2/narrowing.html#the-in-operator-narrowing.

Reply:
I can look into this more tomorrow, but this is consistent with our other calls to `openCommandResponse` for tensorboard

Reply:
The function which we've called to get this, `openOrCreateTensorBoard` (in apiConfig.ts), has a response type `Promise<Type.CommandResponse>`. So in this action's if statement, we can expect a `CommandResponse`.